light/light_test.go

```diff
@@ -167,3 +167,76 @@ func waitForBlock(ctx context.Context, p provider.Provider, height int64) (*type
 		}
 	}
 }
+func TestClientStatusRPC(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	conf, err := rpctest.CreateConfig(t.Name())
+	require.NoError(t, err)
+
+	// Start a test application
+	app := kvstore.NewApplication()
+
+	_, closer, err := rpctest.StartTendermint(ctx, conf, app, rpctest.SuppressStdout)
+	require.NoError(t, err)
+	defer func() { require.NoError(t, closer(ctx)) }()
+
+	dbDir, err := os.MkdirTemp("", "light-client-test-status-example")
+	require.NoError(t, err)
+	defer os.RemoveAll(dbDir)
```

Comment:
```suggestion
	t.Cleanup(func() { os.RemoveAll(dbDir) })
```

Reply:
@creachadair This is copy paste from the previous test that I did not write. I can change this in other places as well. In fact the two previous tests have more-less the same code at the beginning. Do you think refactoring them to call one function would introduce problems ?

Reply:
I was just looking at the new code in isolation. The reason I suggested this is that defers run in reverse order, so the database removal would happen before the close check above. It's possible the closer will fail if the database directory is deleted before it runs, though maybe not.

The test `Cleanup` hooks run after the test returns (and hence, after all the defers have completed).

That said: This isn't a big issue, it's just something that stuck out to me.

Reply:
Thanks for the clarification, very much appreciated :) I have changed it only here for now.